prettier.config.js

```diff
@@ -1,3 +1,11 @@
 module.exports = {
     ...require("./node_modules/jaz-ts-utils/prettier.config"),
-};
\ No newline at end of file
+    overrides: [
+        {
+            "files": "src/**/*.vue",
+            "options": {
+                "printWidth": 140
+            }
+        }
+    ]
```

Comment:
ha, I couldn't settle on a good printWidth to use when I first setup prettier. I messed around with different values but it always looked weird in some places and fine in others. I'm not really that fussed tbh, so happy to go with this